src/script/common/40-yii-migration.sh

```
@@ -0,0 +1,18 @@
+# =============================================================================
+# Run migrations
+# =============================================================================
+yii_run_migrations() {
+    [[ "${YII_RUN_MIGRATIONS:-false}" != "true" ]] && return
+    [[ ! -f "/var/www/app/yii" ]] && { log WARNING "Yii console not found"; return; }
+    [[ "${YII_ENV:-}" == "test" ]] && { log INFO "Test environment, skipping migrations"; return; }
+
+    log INFO "Running database migrations..."
+    cd /var/www/app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)